.drone.yml

```
@@ -102,11 +102,16 @@ steps:
       - make test-frontend
     depends_on: [lint-frontend]
 
+  - name: generate-frontend
+    image: golang:1.19
+    commands:
+      - make generate-frontend
+
   - name: build-frontend
     image: node:18
     commands:
       - make frontend
```

```suggestion
      - make frontend
      depends_on: [deps-frontend]
```

looks that's necessary. https://drone.gitea.io/go-gitea/gitea/60274/1/12

---

ah yes, I see the issue.

---

fixed